CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+* Step 5: Commit all the changes (Write commit message as "Small Message")
+    ```
+    git commit -m "Write a meaningfull but small commit message"
+    ```
+* Step 6: Push the changes for review
+    ```
+    git push origin <branch-name>
+    ```
+* Step 7: Create a PR on Github. (Don't just hit the create a pull request button, you must write a PR message to clarify why and what are you contributing)
+
+
+### 🔨Note:
+
+> - Do not edit/delete someone else's script in this repository. You can only insert new scripts in this repository.
+
+  > - Give a meaningful name to whatever file or folder you are adding.
+  > - To state an example, folder name should be as "Images" and file name should be "front_image.jpg"
+
+
+## 🔑Guidelines
+
+1. Welcome to this repository, if you are here as open source program participant/contributor.
+2. Participants / contributors have to **comment** on issues they would like to work on, and mentors or the PA will assign you.
+3. Issues will be assigned on a **first-come, first-serve basis.**
+4. Participants / Contributors can also **open their issues** using [issue_template](https://github.com/arpitghura/typing-test/tree/main/.github/ISSUE_TEMPLATE),
+but it needs to be verified and labelled by a mentor or PA. Please discuss with the team once before opening your issues. We respect all your contributions, whether 
+it is an Issue or a Pull Request.
+6. When you raise a issue, make sure you get it assigned to you, before you start working on that project.
+7. Each participant / contributor will be **assigned 1 issue (max)** at a time to work.
+8. Participants are expected to follow **project guidelines** and [**coding style**](https://javascript.info/coding-style). **Structured code** is one of our top priority.
+9. Try to **explain your approach** to solve any issue in the comments. This will increase the chances of you being assigned.
+10. Don't create issues that are **already listed**.
+11. Please don't pick up an issue already assigned to someone else. Work on the issues after it gets **assigned to you**.
+12. Make sure you **discuss issues** before working on the issue.
+13. Pull requests will be merged after being **reviewed** by a mentor or PA.
+14. It might take **a day or two** to review your pull request. Please have patience and be nice.
+15. Always create a pull request from a **branch** other than `main`.
+16. Participants / contributors have to complete issues before the decided Deadline. If you fail to make a PR within the deadline, then the issue will be assigned to 
+another person in the queue.
+17. While making PRs, don't forget to **add a description** of your work.
+18. Include issue number (Fixes:issuenumber) in your commit message while creating a pull request.
+19. Make sure your solution to any issue is better in terms of performance and other parameters in comparison to the previous work.
+20. We all are here to learn. You are allowed to make mistakes. That's how you learn, right!.
```

Comment:
```suggestion
8. Participants are expected to follow **project guidelines** and [**coding style**](https://javascript.info/coding-style). **Structured code** is one of our top priorities.
9. Try to **explain your approach** to solve any issue in the comments. This will increase the chances of you being assigned.
10. Don't create issues that are **already listed**.
11. Please don't pick up an issue already assigned to someone else. Work on the issues after it gets **assigned to you**.
12. Make sure you **discuss issues** before working on the issue.
13. Pull requests will be merged after being **reviewed** by a mentor or PA.
14. It might take **a day or two** to review your pull request. Please have patience and be nice.
15. Always create a pull request from a **branch** other than `main`.
16. Participants / contributors have to complete issues before the decided Deadline. If you fail to make a PR within the deadline, then the issue will be assigned to 
another person in the queue.
17. While making PRs, don't forget to **add a description** of your work.
18. Include the issue number (Fixes:issuenumber) in your commit message while creating a pull request.
19. Make sure your solution to any issue is better in terms of performance and other parameters in comparison to the previous work.
20. We all are here to learn. You are allowed to make mistakes. That's how you learn, right!.
```